styles/tables/mod.css

```diff
@@ -1,44 +1,59 @@
+/* Modernized table styles */
+
 table {
-  display: block;
+  width: 100%;
```

Comment:
The `display: block` is a trick to make overflowable table without the need of using a wrapper div when width/parent width is set, so I feel like it's better to keep it.